### ISSUE TITLE
create-directus-extension throws "Named export 'parse' not found" error

### ISSUE THREAD

**Issue by D3VL-Jack** [directus/directus#22835]
### Describe the Bug

When running `npx create-directus-extension@latest` as found throughout the documentation[^1] the following error is immediately thrown 

```log
file:///{{REDACT}}/node_modules/rollup-plugin-esbuild/dist/index.mjs:11
import { parse } from "jsonc-parser";
         ^^^^^
SyntaxError: Named export 'parse' not found. The requested module 'jsonc-parser' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'jsonc-parser';
const { parse } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)

Node.js v20.12.2
```

Another user in the Discord server[^2] reported this happening not when installing, but when building using `npm run build`

[^1]: [modules-build-landing-page.html#install-dependencies](https://docs.directus.io/guides/extensions/modules-build-landing-page.html#install-dependencies)
[^2]: [Discord message link](https://discord.com/channels/725371605378924594/1151320093565927504/1254851831260250173)

### To Reproduce

Open a new terminal and run `npx create-directus-extension@latest`

Environment:
- Win 11 Powershell
- Node v20.12.2
- NPM 10.5.0

### Directus Version

@latest

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by rijkvanzanten** [comment]
I believe this to be caused a breaking change in a minor patch in the nested dependency `jsonc-parser`. Tracking that in https://github.com/microsoft/node-jsonc-parser/issues/91

**Comment by paescuj** [comment]
A fix has been released in [jsonc-parser@3.3.1](https://github.com/microsoft/node-jsonc-parser/releases/tag/v3.3.1).

`create-directus-extension` is working again:
```console
❯ npx create-directus-extension@latest
This utility will walk you through creating a Directus extension.

? Choose the extension type (Use arrow keys)
❯ interface 
  display 
  layout 
  module 
  panel 
  theme 
  hook 
(Move up and down to reveal more choices)
```
